driver-core/src/main/com/mongodb/internal/connection/InternalStreamConnection.java

```diff
@@ -95,6 +97,9 @@
  */
 @NotThreadSafe
 public class InternalStreamConnection implements InternalConnection {
+    // otherwise, would not exist
+    @VisibleForTesting(otherwise = VisibleForTesting.AccessModifier.PRIVATE)
```

Comment:
Removed, but `otherwise = WOULD_NOT_EXIST` differs from private, in that it indicates that the method would be removed entirely (not merely made private) if tests did not need it.

Reply:
`@VisibleForTesting(otherwise = PRIVATE)` means that the program element may be used only by tests and by the class that declares it. If you remove tests that use the program element, and the element is not used by the declaring class, the element becomes dead code and will be removed. If the element is still used by the declaring class, then it will stay. There is no reason to document a program element as "must not be used by the declaring class" (which is what `otherwise = WOULD_NOT_EXIST` means).